core/app/queries/workarea/featured_categorization.rb

```diff
@@ -39,9 +39,19 @@ def category_changesets_affecting_current_release
       return [] if Release.current.blank?
 
       FeaturedProducts
-        .changesets(@product.id)
+        .changesets(*product_ids)
         .where(releasable_type: Catalog::Category.name)
         .in(release_id: Release.current.preview.releases.map(&:id))
     end
+
+    private
+
+    def product_ids
```

Comment:
If we're going to change this class to support multiple product IDs, and we don't really use the product model anyways, can we go all the way and update the initializer to accept multiple product IDs?

Reply:
Its not meant for multiple products though, just that with some plugins, a single product can sometimes be represented with variations of it's ID. I'll bring this up in the standup